### PR TITLE
Fix #94: Added test sketch, README.md and git-repos.dat for Catena 4801

### LINKED ISSUE #94: Add simple test sketch for Catena 4801
Add a simple test sketch, git-repos.dat and README.md for Catena 4801.

### PR DESCRIPTION
**Pull request by dhineshkumarmcci**
Added a simple test sketch, README.md and git-repos.dat for Catena 4801.

### CODE REVIEW

File: catena4801_simple/README.md [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+```
+/* Modbus slave parameters */
+u8addr = 65;			// current target device: 01
+u8fct = 03;			// Function code
+u16RegAdd = 00;			// start address in device
+u16CoilsNo = 13;		// number of elements (coils or registers) to read
+host.begin(&mySerial, 9600);	// baud-rate at 9600
+```
+
+Connect your modbus slave�s A and B terminal to Catena 4801�s A and B terminal (JP3).
```
Comment (chwon64) [inline catena4801_simple/README.md]: There is special characters after slave and 4801.

File: catena4801_simple/README.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+
+Load the sketch into the Catena using `Sketch`>`Upload` and move on to provisioning.
+
+## Set the identity of your Catena 4801
+
+This can be done with any terminal emulator, but it's easiest to do it with the serial monitor built into the Arduino IDE or with the equivalent monitor that's part of the Visual Micro IDE. It can also be done usign Tera Term.
+
+### Check platform amd serial number setup
+
+![Newline](./assets/serial-monitor-newline.png)
```
Comment (chwon64) [inline catena4801_simple/README.md]: It looks link is wrong or file is missing.

File: catena4801_simple/README.md (hunk 1 above)
Comment (dhineshkumarmcci) [reply catena4801_simple/README.md]: Made the required changes

File: catena4801_simple/README.md (hunk 2 above)
Comment (dhineshkumarmcci) [reply catena4801_simple/README.md]: Added the file to the directory

**Comment by svelmurugan92** [comment]
@ChaeHee Thank you for the review. 
@terrillmoore We will merge the pull request.

**Pull request merged**